examples/nwc/client/subscribe.ts

```
@@ -14,10 +14,12 @@ const client = new NWCClient({
   nostrWalletConnectUrl: nwcUrl,
 });
 
-const onNotification = (notification) =>
+const onNotification = (notification: Nip47Notification) =>
   console.info("Got notification", notification);
```

ESLint

[error] Unexpected console statement.

(no-console)